### DISCUSSION TITLE
List for adding Apps

### DISCUSSION THREAD

**Discussion by paulp2022** [jamf/Setup-Manager#36] [Q&A] [answered]
Hi All,

New Jamf Pro user and have just got this up and running as per the instructions.

Google Chrome is installing on my test Laptop.

The issue I have is that I cant seem to get any other app installed.

Is there a list of Apps we can install via this system and gives the settings we nee to enter?

For example For 1 password I have tried Label = 1Password, Installomator label = 1Passwordpkg and also 1password8.sh as per https://github.com/Installomator/Installomator/blob/dev/fragments/labels/1password8.sh but neither work.

Thanks for any help on this

**Comment by elior-esh** [comment]
Hi!

You can check the Installomator labels here : https://github.com/Installomator/Installomator/blob/main/Labels.txt

So, for 1Password installation your schema should look like this but with proper indentation
`<dict>
      <key>label</key>
      <string>1Password 8</string>
      <key>icon</key>
      <string>ICON_PATH</string>
      <key>installomator</key>
      <string>1password8</string>
</dict>`


Hope that helps.

**Comment by paulp2022** [comment]
Hi [elior-esh](https://github.com/elior-esh)

Thanks for replying, however this is supposed to be simplified as per the attached image, could you cut it down please so that I know what to put into the attached for 1password8
<img width="899" alt="Screenshot 2024-09-26 at 15 40 39" src="https://github.com/user-attachments/assets/bace513c-9f41-4278-b906-2b6485962a78">

**Reply by elior-esh** [reply, marked as answer]
Sure. 
Item 1 = Left as is - Installomator
Action Label = 1Password 8 (or however you want this to be displayed in the JSM gui)
Action Icon Source = Choose which icon you want to display and how. Let me know if you need guidance with this - most common way is to upload an icon to a Self Service policy and get the URL from there.
Installomator Label = **1password8** (per the URL I attached in my previous comment with the full labels list)

That should do the trick

**Comment by paulp2022** [comment]
Ohhh thank you so much, that has worked so I can go ahead and set up everything for each department 🥇 

I guess my mistake was adding the pkg or .sg to the end of the Installomator Label 

If you could give me a little guidance on the Action Icon Source that would be great but it's not important at this stage.

Thank you so much

**Comment by scriptingosx** [comment]
The `icon` can be a local image file (usually one that you would have installed with a separate branding package in the Prestage before running Setup Manager, a URL (Setup Manager will download the image), a SF Symbol, or an App icon (but again, the app would need to be installed before Setup Manager runs, so only system apps work reliably this way.

https://github.com/Jamf-Concepts/Setup-Manager/blob/main/ConfigurationProfile.md#icon-sources

You can also upload an icon to a Self Service policy in Jamf and use that URL:

https://github.com/Jamf-Concepts/Setup-Manager/blob/main/Docs/FAQ.md#how-can-i-use-the-icon-for-an-app-before-the-app-is-installed

**Comment by paulp2022** [comment]
Perfect, thank you both so much, hopefully this will help other people as well
